### PR TITLE
LPS-133397 | Master

### PR DESCRIPTION
**Pull request by ThalitaCeli**

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase [hunk 1]
```diff
@@ -588,6 +588,91 @@ definition {
...
+			PageEditor.expandFragmentCollection(collectionName = "Basic Components");
+
+			DragAndDrop.javaScriptDragAndDropToObject(
+				key_collectionName = "Basic Components",
+				key_columnNumber = "1",
+				key_fragmentName = "Heading",
+				key_position = "2",
+				locator1 = "PageEditor#FRAGMENTS_AND_WIDGETS_TAB_FRAGMENT",
+				locator2 = "PageEditor#GRID_COLUMN",
+				value1 = "");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase:622]: Use PageEditor.addElementToColumn to simplify

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase [hunk 2]
```diff
@@ -588,6 +588,91 @@ definition {
...
+			PageEditor.expandFragmentCollection(collectionName = "Basic Components");
+
+			DragAndDrop.javaScriptDragAndDropToObject(
+				key_collectionName = "Basic Components",
+				key_columnNumber = "2",
+				key_fragmentName = "Button",
+				key_position = "2",
+				locator1 = "PageEditor#FRAGMENTS_AND_WIDGETS_TAB_FRAGMENT",
+				locator2 = "PageEditor#GRID_COLUMN",
+				value1 = "");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase:635]: Use PageEditor.addElementToColumn to simplify

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase [hunk 3]
```diff
@@ -588,6 +588,91 @@ definition {
...
+			PageEditor.expandFragmentCollection(collectionName = "Basic Components");
+
+			DragAndDrop.javaScriptDragAndDropToObject(
+				key_collectionName = "Basic Components",
+				key_columnNumber = "3",
+				key_fragmentName = "Paragraph",
+				key_position = "2",
+				locator1 = "PageEditor#FRAGMENTS_AND_WIDGETS_TAB_FRAGMENT",
+				locator2 = "PageEditor#GRID_COLUMN",
+				value1 = "");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase:648]: Use PageEditor.addElementToColumn to simplify

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase [hunk 4]
```diff
@@ -588,6 +588,91 @@ definition {
...
+		}
+
+		task ("Assert if the ellipsis button is visible on the Heading fragment") {
+			PageEditor.clickConfigurableField(fragmentName = "Heading");
+
+			AssertElementPresent(
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase:654]:
```suggestion
			AssertVisible(
```

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase [hunk 5]
```diff
@@ -588,6 +588,91 @@ definition {
...
+				key_fragmentName = "Heading",
+				locator1 = "Fragment#FRAGMENT_HEADER_ELLIPSIS");
+		}
+
+		task ("Assert if the ellipsis button is visible on the Button fragment") {
+			PageEditor.clickConfigurableField(fragmentName = "Button");
+
+			AssertElementPresent(
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase:662]:
```suggestion
			AssertVisible(
```

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase [hunk 6]
```diff
@@ -588,6 +588,91 @@ definition {
...
+				key_fragmentName = "Button",
+				locator1 = "Fragment#FRAGMENT_HEADER_ELLIPSIS");
+		}
+
+		task ("Assert if the ellipsis button is visible on the Paragraph fragment") {
+			PageEditor.clickConfigurableField(fragmentName = "Paragraph");
+
+			AssertElementPresent(
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase:670]:
```suggestion
			AssertVisible(
```

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase [hunk 7]
```diff
@@ -588,6 +588,91 @@ definition {
...
+				key_collectionName = "Basic Components",
+				key_columnNumber = "3",
+				key_fragmentName = "Paragraph",
+				key_position = "2",
+				locator1 = "PageEditor#FRAGMENTS_AND_WIDGETS_TAB_FRAGMENT",
+				locator2 = "PageEditor#GRID_COLUMN",
+				value1 = "");
+		}
+
+		task ("Assert if the ellipsis button is visible on the Heading fragment") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase:651]: View the ellipsis button is visible on topper of Heading fragment

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase [hunk 8]
```diff
@@ -588,6 +588,91 @@ definition {
...
+			PageEditor.clickConfigurableField(fragmentName = "Heading");
+
+			AssertElementPresent(
+				key_fragmentName = "Heading",
+				locator1 = "Fragment#FRAGMENT_HEADER_ELLIPSIS");
+		}
+
+		task ("Assert if the ellipsis button is visible on the Button fragment") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase:659]: View the ellipsis button is visible on topper of Button fragment

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase [hunk 9]
```diff
@@ -588,6 +588,91 @@ definition {
...
+			PageEditor.clickConfigurableField(fragmentName = "Button");
+
+			AssertElementPresent(
+				key_fragmentName = "Button",
+				locator1 = "Fragment#FRAGMENT_HEADER_ELLIPSIS");
+		}
+
+		task ("Assert if the ellipsis button is visible on the Paragraph fragment") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase:667]: View the ellipsis button is visible on topper of Paragraph fragment

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase [hunk 10]
```diff
@@ -588,6 +588,91 @@ definition {
 		}
 	}
 
+	@description = "This is for LPS-130083. View the ellipsis button on the top bar when the fragment is inside the nested grid."
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase:591]: View the ellipsis button on the topper of fragment when the fragment is inside a nested Grid.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase [hunk 11]
```diff
@@ -588,6 +588,91 @@ definition {
...
+	@priority = "4"
+	test ViewEllipsisButtonInsideNestedGrid {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase:593]: ViewEllipsisButtonOnTheTopperOfFragmentInsideNestedGrid

**Comment by Tim-Cao** [comment]
Hi @ThalitaCeli, Could you modify case according to my comments? Thanks

**Comment by ThalitaCeli** [comment]
> Hi @ThalitaCeli, Could you modify case according to my comments? Thanks

Hi @Tim-Cao
Done.
If you need any more changes let me know OK?

Thank you.

**Comment by Tim-Cao** [comment]
Hi @ThalitaCeli, see changes https://github.com/liferay-echo/liferay-portal/compare/947b6c484e4bd077da40c1730ae8c4a8aa0304a1...934b006cbc35bc1d6b651eca06cf14c6f92bee0f
https://github.com/liferay-echo/liferay-portal/pull/4987

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**